Tasks/DownloadGitHubReleaseV0/task.json

```diff
@@ -10,7 +10,7 @@
   "demands": [],
   "version": {
     "Major": 0,
-    "Minor": 148,
+    "Minor": 149,
```

Comment:
If these changes are to be checked-in in M148 we should only change the patch version

Reply:
We won't make it 149 as lova suggested. It will remain 148 most probably unless my check in is delayed.

Reply:
This should remain as M148.